src/Extensions/LanguageConfigurationLoader.re

```diff
@@ -1,11 +1,18 @@
 open Oni_Core;
+open Oni_Core.Utility;
+
+module Log = (val Log.withNamespace("Oni2.LanguageConfigurationLoader"));
 
 type t = LazyLoader.t(LanguageConfiguration.t);
 
 let create = languageInfo => {
   let loadFunction = language => {
+    Log.tracef(m => m("Loading configuration for language: %s", language));
```

Comment:
Is this going to be run so frequently that it would be overwhelming with `debug` logging?

Reply:
I don't think so - that seems like a reasonable upgrade. Right now, it's on user input, but wouldn't be the bottleneck (and only happens once). I'll make that change